Mathlib/Topology/Metrizable/CompletelyMetrizable.lean

```diff
@@ -0,0 +1,176 @@
+/-
+Copyright (c) 2025 Etienne Marion. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Etienne Marion
+-/
+import Mathlib.Topology.MetricSpace.Gluing
+import Mathlib.Topology.Metrizable.Uniformity
+
+/-!
+# Completely metrizable spaces
+
+A topological space is completely metrizable if one can endow it mith a `MetricSpace` structure
+which makes it complete and gives the same topology. This typeclass allows to state theorems
+which do not require a `MetricSpace` structure to make sense without introducing such a structure.
+It is in particular useful in measure theory, where one often assumes that a space is a
+`PolishSpace`, i.e. a separable and completely metrizable space. Sometimes the separability
+hypothesis is not needed and the right assumption is then `CompletelyMetrizableSpace`.
+
+## Main definition
+
+* `CompletelyMetrizableSpace X`: A topological space is completely metrizable if there exists a
```

Comment:
Could you rename it to `IsCompletelyMetrizableSpace X`? I know we are not coherent, especially in the topological part of the library, but this looks strictly better to me.